### DISCUSSION TITLE
1981-10-17-saturday-night-is-gala-night/

### DISCUSSION THREAD

**Discussion by giscus** [FanzOfLenaZavaroni/fanzoflenazavaroni.github.io#34] [General]
# 1981-10-17-saturday-night-is-gala-night/

A concert of British music from the Royal Albert Hall, London in association with the Music Publishers Association of Great Britain to celebrate their centenary.

https://fanzoflenazavaroni.github.io/1981-10-17-saturday-night-is-gala-night/

**Comment by StephenMolloy1** [comment]
He sang a couple of his songs, including "Maybe it's because I'm a Londoner", starting at about 5:40 mins in.
https://www.youtube.com/watch?v=UblmGfMCl1o&ab_channel=RadioRecollections

**Reply by FanzOfLenaZavaroni** [reply]
Thanks for the info, and link. I have added the details to this page.
